lerobot/common/policies/diffusion/modeling_diffusion.py

```diff
@@ -111,12 +111,12 @@ def select_action(self, batch: dict[str, Tensor]) -> Tensor:
         Schematically this looks like:
             ----------------------------------------------------------------------------------------------
             (legend: o = n_obs_steps, h = horizon, a = n_action_steps)
-            |timestep            | n-o+1 | n-o+2 | ..... | n     | ..... | n+a-1 | n+a   | ..... |n-o+1+h|
-            |observation is used | YES   | YES   | YES   | NO    | NO    | NO    | NO    | NO    | NO    |
+            |timestep            | n-o+1 | n-o+2 | ..... | n     | ..... | n+a-1 | n+a   | ..... | n-o+h |
+            |observation is used | YES   | YES   | YES   | YES   | NO    | NO    | NO    | NO    | NO    |
             |action is generated | YES   | YES   | YES   | YES   | YES   | YES   | YES   | YES   | YES   |
             |action is used      | NO    | NO    | NO    | YES   | YES   | YES   | NO    | NO    | NO    |
             ----------------------------------------------------------------------------------------------
-        Note that this means we require: `n_action_steps < horizon - n_obs_steps + 1`. Also, note that
+        Note that this means we require: `n_action_steps < horizon - n_obs_steps`. Also, note that
```

Comment:
I thought about this more and realized it should be.

```suggestion
        Note that this means we require: `n_action_steps <= horizon - n_obs_steps + 1`. Also, note that
```

Reply:
example:
n_obs_steps = 2
horizon = 4
n_action_steps is max 3  = horizon - n_obs_steps + 1
```
                | n-1 | n  | n + 1 | n + 2
obs          |  Y   | Y  |    N   |    N
act used  |  N   | Y |    Y    |    Y
```